### ISSUE TITLE
molvs hugs on some molecules

### ISSUE THREAD

**Issue by sergsb** [mcs07/MolVS#21]
Hi,
I found that `molvs` hugs on some molecules i.e.
```
from molvs import standardize_smiles, __version__
print(__version__)
standardize_smiles("[Na].C(C=C)C1(C(NC(NC1=O)=S)=O)CC1=CC=CC=C1")
```
returns `0.0.9 ` and then falls into infinite loop

**Comment by mcs07** [comment]
Please try updating to version 0.1.0, I think this issue has been fixed.

**Comment by mcs07** [comment]
Fixed in  #18
